internal/storage/bucket_handle.go

```diff
@@ -509,6 +509,21 @@ func (b *bucketHandle) getStorageLayout() (*controlpb.StorageLayout, error) {
 	return stoargeLayout, err
 }
 
+func (b *bucketHandle) GetFolder(ctx context.Context, folderName string) (*controlpb.Folder, error) {
+	var callOptions []gax.CallOption
+
+	folder, err := b.controlClient.GetFolder(ctx, &controlpb.GetFolderRequest{
+		Name: "projects/_/buckets/" + b.bucketName + "/folders/" + folderName,
+	}, callOptions...)
+
+	if err != nil {
```

Comment:
Should we just return `folder, err` and catch the err where ever we call this method like `GetFolder: error` ?

Reply:
done